src/components/Job.jsx

```diff
@@ -54,11 +66,44 @@ const Job = ({ job }) => {
         if (diffMs < 0) return true
     }
 
+    const UrlbyOrg = job.details?.match(/(https?:\/\/[^\s]+)/g)[0];
+
+
+
+    const [showTooltip, setShowTooltip] = useState("copy"); // Initial state for tooltip
+    const [copied, setCopied] = useState(false); // State to manage emoji
+
+    const copyToClipboard = (text) => {
+        navigator.clipboard.writeText(text)
+            .then(() => {
+                setCopied(true); // Change the emoji state to "copied"
+                setShowTooltip("Copied!"); // Change tooltip text
+                // Set a timeout to revert back to the original emoji and tooltip text
+                setTimeout(() => {
+                    setCopied(false);
+                    setShowTooltip("copy"); // Reset tooltip to original
+                }, 1000); // Change back after 1 second
+            })
+            .catch((err) => {
+                console.error('Failed to copy: ', err);
+            });
+    };
+
     return (
         <>
 
             <section className="bg-indigo-50">
-                <div className="container m-auto py-10 px-6">
+                <div className=" w-full h-[450px]">
+                    {/* this is for event banner image */}
+
+                    <img className="object-fill w-full max-h-[450px]" src={`${job.images[0]}`} alt="" />
```

Comment:
Handle if image is not uploaded.
Agar bola hai to dekhte hai, nhi to abhi ke liye rehde de.
(styling shi karna padega)

yeh ratio mujhe mila tha uski website se of img 
`img[Attributes Style] {
    width: 956px;
    aspect-ratio: auto 956 / 298;
    height: 298px;
}`

[Link of banner img](https://www.proelevate.in/_next/image?url=https%3A%2F%2Fcdn.proelevate.in%2FprofileImages%2F1725174568476_arnu6h2nw9.jpeg&w=3840&q=75)